subsys/spm/spm.c

```diff
@@ -135,8 +135,21 @@ static void spm_config_nsc_flash(void)
 }
 #endif /* CONFIG_ARM_FIRMWARE_HAS_SECURE_ENTRY_FUNCS */
 
+extern int spm_secure_services_init(void);
```

Comment:
Could we move this prototype into secure_services.h and include it that header from here?

If not, can we move this prototype into another header file?

Also, drop the 'extern' keyword, it does nothing to the compiler, and could confuse a programmer.

Reply:
I'd like secure_services.h to contain only functions available from the non-secure side, is spm_internal.h ok?

Reply:
yes

Reply:
Done